quay-enterprise/ldap-setup.md

```diff
@@ -0,0 +1,40 @@
+# LDAP Authentication Setup for Quay Enterprise
+
+The Lightweight Directory Access Protocol (LDAP) is an open, vendor-neutral, industry standard application protocol for accessing and maintaining distributed directory information services over an Internet Protocol (IP) network. Quay Enterprise supports using LDAP as an identity provider.
+
+## Before you start
+
+Quay Enterprise LDAP setup workflow requires user that’s setting up the configuration to pre-exist in LDAP directory. Before attempting the setup make sure that you are logged in as a superuser that matches a user in LDAP. In order to do so, Navigate to the SuperUser panel (ex: http(s)://quay.enterprise/superuser) and click on “Create User” button to create a new User. Make sure to create a user that matches username/email syntax in LDAP. 
+
+Once the user is created, click on the Settings icon next to the user and choose “Make Superuser” option. For ease of troubleshooting, set the User password to LDAP password. 
+
+<img src="img/superuser-user-settings.png" class="img-center" alt="Enable GitHub Authentication"/> 
+
+You will be prompted to restart the container once the new user is created. Restart Quay Enterprise container and log in to the Superuser panel as the user that’s been previously created. 
+
+## Setup LDAP Configuration
+
+<img src="img/authentication-ldap.png" class="img-center" alt="Enable GitHub Authentication"/>
+
+Navigate to the Superuser panel and navigate to settings section. Locate the Authentication section and select “LDAP” from the drop-down menu. 
+
+Enter LDAP configuration fields as required.
+
+<img src="img/authentication-ldap-details.png" class="img-center" alt="Enable GitHub Authentication"/>
+
+### Few tips for LDAP configuration:
+
+- LDAP URI must be in ldap:// or ldaps:// syntax. Typing a URI with ldaps:// prefix will surface the option to provide custom SSL certificate for TLS setup
+- User Relative DN is relative to BaseDN (ex: ou=NYC not ou=NYC,dc=example,dc=org)
+- Logged in Username must exist in User Relative DN
+- You can enter multiple “Secondary User Relative DNs” if there are multiple Organizational Units where User objects are located at. (ex: ou=Users,ou=NYC and ou=Users,ou=SFO). Simply type in the Organizational Units and click on Add button to add multiple RDNs
+- sAMAccountName is the UID attribute for against Microsoft Active Directory setups
+
+Once the configuration is completed, click on “Save Configuration Changes” button. You will be prompted to login with ***LDAP credentials***. 
+
+<img src="img/authentication-ldap-success.png" class="img-center" alt="Enable GitHub Authentication"/>
+
+## Common Issues
+
+- "Invalid credentials" - Administrator DN or Administrator DN Password values are incorrect
```

Comment:
This might be better as a set of paragraphs with the messages bold:

**"Invalid credentials"***
The Administrator DN or Administrator DN Password values are incorrect

**"Verification of superuser %USERNAME% failed: Username not found The user either does not exist in the remote authentication system OR LDAP auth is misconfigured."**
Quay can connect to the LDAP server via Username/Password specified in the Administrator DN fields however cannot find the current logged in user with the UID Attribute or Mail Attribute fields in the User Relative DN Path. Either current logged in user does not exist in User Relative DN Path, or Administrator DN user do not have rights to search/read this LDAP path.